src/utils/hints/contractsHints.js

```diff
@@ -3,4 +3,18 @@ export const contractsHints = {
   contractCreated: 'Keyblock height and exact date and time when the smart contract was created.',
   contractTxHash: 'The transaction hash of a ContractCreateTx that was executed to create the smart contract on the blockchain.',
   contractCreator: 'Account that created the smart contract.',
+  bytecode: 'Low-level representation of the smart contract that is generated when the contract\'s source code is compiled.',
+  contractsAccount: 'The address where the smart contract is located. To be used if a smart contract is already deployed.',
+  contractsAccountBalance: 'Smart contract account balance that is needed to execute "payable" functions.',
+  contractCalls: 'Amount of calls to the smart contract.',
+  apiLinks: 'Links to sources of the smart contract data.',
+  callTransactionsHash: 'Low-level representation of the smart contract that is generated when the contract\'s source code is compiled.',
+  callTransactionsTime: 'Estimated date, time and block height when the smart contract was called.',
+  callTransactionsCaller: 'Account that called the smart contract.',
+  callTransactionsEntryPoint: 'Name of the smart contract function that was called.',
+  eventsCallTransaction: 'Unique identifier of the ContractCallTx transaction which emitted the event.',
+  eventsCreated: 'Estimated date, time and block height when the smart contract had emitted the event.',
+  eventsName: 'Name of the emitted event.',
+  eventsArguments: 'The arguments of the emitted event.',
+  data: 'The content of the emitted event.',
```

Comment:
is data really relevant / used ? 🤔  aren't arguments the data emitted?

Reply:
Afaig arguments ar data (parameters) passed to entrypoint